packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1101,42 +1104,31 @@ describe("SharedTree", () => {
 				scope: "shared tree undo tests",
 				libraries: [leaf.library],
 			});
-			const stringSequenceSchema = sb.fieldNode(
-				"stringList",
-				FlexFieldSchema.create(FieldKinds.sequence, [leaf.string]),
-			);
-			const schema = sb.intoSchema(
-				FlexFieldSchema.create(FieldKinds.sequence, [stringSequenceSchema]),
-			);
-
-			function makeTwoEdits(
-				peer: CheckoutFlexTreeView<typeof schema.rootFieldSchema>,
-			): Revertible[] {
-				const undos: Revertible[] = [];
-				const unsubscribe = peer.checkout.events.on(
-					"newRevertible",
-					(revertible: Revertible) => {
-						if (revertible.kind !== RevertibleKind.Undo) {
-							revertible.retain();
-							undos.push(revertible);
-						}
-					},
-				);
+			const innerListSchema = FlexFieldSchema.create(FieldKinds.sequence, [leaf.string]);
+			const innerListNodeSchema = sb.fieldNode("stringList", innerListSchema);
+			const outerListSchema = FlexFieldSchema.create(FieldKinds.sequence, [
+				innerListNodeSchema,
+			]);
+			const schema = sb.intoSchema(outerListSchema);
+			const config = {
+				schema,
+				allowedSchemaModifications: AllowedUpdateType.None,
+				initialTree: [["a"]],
+			};
 
-				const outerList = peer.flexTree;
-				const innerList = (outerList.at(0) ?? assert.fail()).content;
-				innerList.insertAtEnd("b");
-				innerList.insertAtEnd("c");
+			type TreeView = CheckoutFlexTreeView<typeof outerListSchema>;
 
-				unsubscribe();
-				return undos;
+			interface Peer extends ConnectionSetter {
+				readonly view: TreeView;
+				readonly outerList: FlexTreeTypedField<typeof outerListSchema>;
+				readonly innerList: FlexTreeTypedField<typeof innerListSchema>;
```

Comment:
storing these seem kind of unnecessary since they're only used once each

Reply:
oh, twice actually, which I think is still fine since we don't have to dig as deep anymore

Reply:
They also make debugging a little easier since you can just inspect them through hover.